sotodlib/hwp/g3thwp.py

```diff
@@ -90,8 +90,9 @@ def __init__(self, config_file=None):
         # 1: positive rotation direction, -1: negative rotation direction
         self._force_quad = int(self.configs.get('force_quad', 0))
         if np.abs(self._force_quad) > 1:
-            logger.error("force_quad in config file must be 0 or 1 or -1")
-            sys.exit(1)
+            logger.warning("force_quad in config file must be 0 or 1 or -1")
+            if self._force_quad > 1: self._force_quad = 1
```

Comment:
PEP8 standards say no single line ifs / elses like this. 
```
if conditional:
    do_things()
else:
    do_other_things()
```

Reply:
Applying autopep8 fixes this.